synapse/storage/roommember.py

```diff
@@ -35,6 +36,11 @@
 )
 
 
+ProfileInfo = namedtuple(
```

Comment:
Maybe add a comment on why we use a namedtuple for this that explains how much memory we save by doing so.